CONTRIBUTING.md

```diff
@@ -383,6 +387,12 @@ If you need text-formatting, you <strong>must</strong> use in-line HTML.
 </p></div>
 ```
 
+```md
+<div class="warningbox"><p>
+This class should be used for important information.
+</p></div>
```

Comment:
Didn't the last commit remove the need for the p tag?

Reply:
It removed the need for specifying a new style for this div's paragraph tag. You still need to use the `p` tag for it to inherit the correct styles from the rest of the page. This is what it looks like without the `<p>text</p>`. The margins/padding are missing and the text isn't displayed corectly because I guess it doesn't know that it needs normal html paragraph text-formatting (the spacing around the bold text).
![image](https://user-images.githubusercontent.com/10972040/75621285-242ae380-5b60-11ea-823d-bb71d6b79875.png)

Reply:
All right, thanks for clarifying.